Python/import.c

```diff
@@ -1941,15 +2005,73 @@ import_run_extension(PyThreadState *tstate, PyModInitFunction p0,
         if (cached == NULL) {
             goto error;
```

Comment:
Can we do this without switching back?

Reply:
Yeah, I noticed that too and have already fixed it locally. 😄

Reply:
fixed